streams/src/test/java/org/apache/kafka/streams/integration/AbstractResetIntegrationTest.java

```diff
@@ -255,7 +259,9 @@ public void shouldNotAllowToResetWhenIntermediateTopicAbsent() throws Exception
         final String[] parameters = new String[] {
             "--application-id", appID,
             "--bootstrap-servers", cluster.bootstrapServers(),
-            "--input-topics", NON_EXISTING_TOPIC };
+            "--intermediate-topics", NON_EXISTING_TOPIC,
```

Comment:
Seems to have been a bug that we used `--input-topics` before.

Reply:
+1.

Reply:
Ack.